csharp/Svix/MessageAttempt.cs

```diff
@@ -362,18 +352,18 @@ public MessageAttemptOut GetAttempt(string appId, string attemptId, string messa
         {
             try
             {
-                var lResults = await _messageAttemptApi.ListAttemptsApiV1AppAppIdMsgMsgIdAttemptGetAsync(
+                var lResults = await _messageAttemptApi.V1MessageAttemptListByMsgDeprecatedAsync(
                     appId,
                     messageId,
-                    options?.Iterator,
                     options?.Limit,
+                    options?.Iterator,
```

Comment:
Is there type-checking for the ordering, or do we just hope it's right?
Do I need to double check you there, or have you verified that?

Reply:
There is type checking yes, so things like limit and iterator can't be mixed up because one's an integer and the other is a string, but there are some parameters where mistakes are possible, for example before/after. A double check can't hurt, although I think the chance for mistakes are low because for languages where things are passed by position I checked things one by one.